lectureDL.py

```diff
@@ -407,9 +428,8 @@ def getSubjects(subject_list):
     if user_choice != "":
         # Allows for more flexible input
         selection = [int(x) for x in re.findall(r"\d", user_choice)]
-        # TODO:
-        # Will need to be changed when Subject() is implemented
-        return list(filter(lambda x: any(sel in x for sel in selection),
+        return list(filter(lambda sub: any(sel in sub.num
```

Comment:
Just a self note, this filter isn't the python way, a list comprehension would be. Obviously this was already here so don't change it for this commit.